pkg/traversal/worker.go

```diff
@@ -25,69 +26,54 @@ func (f *fileProcessingError) Cause() error {
 	return f.cause
 }
 
-type workerFile struct {
-	f input.File
+type WorkerInput struct {
```

Comment:
Since `WorkerInput` holds just one string we can turn use strings or a type aliased to string.

Reply:
yeah I thought about this as well, but if we add more context information later on it's not as big of a refactoring. Whatever you prefer.

Reply:
Let's use a type aliased to string.